complete_control/neural/plot_utils.py

```diff
@@ -95,6 +97,12 @@ def plot_rate(time_v, ts, pop_size, buffer_sz, ax, title="", **kwargs):
     ax.set_xlim(left=0, right=time_end)
     ax.set_ylim(bottom=0)
 
+    if (rate_sm.size > 0):      # add return for keep max_value to scale the plot (if rate_sm is not empty)
```

Comment:
mmm i think this line isn't formatted properly. please stick with black formatting
as an alternative, you can move the comment to line 99

```suggestion
    # add return for keep max_value to scale the plot (if rate_sm is not empty)
    if (rate_sm.size > 0):
```